modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/upgrade-catch-all-check/LPD_27178.testjava

```diff
@@ -0,0 +1,18 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+/**
+ * @author Micaelle Silva
+ */
+public class LPD_27178 {
+
+	public void method(Article article) {
+		article.isInTrashExplicitly();
+		article.isInTrashImplicitly();
+	}
```

Comment:
@MicaelleSilvaa since its for more them one model, lets declare another one in the parameters and add in the tests as well, please.